schedule/yast/maintenance/autoyast_create_hdd_maintenance_deregister_s390x.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: autoyast_create_hdd_maintenance_deregister_s390x
```

Comment:
this PR doesn't say why you create only one file for s390x, is expected one per each architecture.

Reply:
> this PR doesn't say why you create only one file for s390x, is expected one per each architecture.

For other architecture we can continue use the schedule/yast/sle_15_sp3/autoyast_create_hdd_maintenance_deregister.yaml, only s390x is different so need create a new file.